index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header>
+            <p class="time"> 9:14 <img class="net" src="C:\Users\Harrison\phone network image.jpg" alt="phone network"></p>
+            <img class="logo" src="C:\Users\Harrison\Downloads\My Logo.png" alt="logo">
+            <img class="navicon" src="https://assets.stickpng.com/images/588a64c6d06f6719692a2d0c.png" alt="nav icon"><br>
```

Comment:
- [ ] I kindly suggest you to wrap your hamburger button with a nav tag. The hamburger button is on the right, and has a `nav` tag.The `nav` tag is for semantics reason, and the right position for convention and user experience for mobile devices.
### example 👇
```
<header>
    <nav class="nav-bar">
        <a href="" class="nav-bar-logo">Mohammed</a>
        <a href="#" class="nav-bar-burger"><img src="rsc/icons/Icon-Menu.svg" alt=""></a>
    </nav>
<header>

```